### PR TITLE
Set default log_level to fatal

### PR DESCRIPTION
**Pull request by armiller**

### CODE REVIEW

File: templates/default/spec_helper.rb.erb [hunk 1]
```diff
@@ -6,17 +6,18 @@ ChefSpec::Coverage.start! { add_filter '<%= cookbook_name %>' }
...
 
 DEBIAN_8_OPTS = {
   platform: 'debian',
   version: '8.4',
-  log_level: :fatal
 }.freeze
+
+Rspec.configure do |config|
+  config.log_level = :fatal
+end
```
Comment (ramereth) [inline templates/default/spec_helper.rb.erb]: I'm getting the following error on a fresh generated cookbook:

```
/tmp/foo/spec/spec_helper.rb:21:in `<top (required)>': uninitialized constant Rspec (NameError)
```

**Comment by armiller** [comment]
Fixed.

File: templates/default/spec_helper.rb.erb [hunk 2]
```diff
@@ -6,17 +6,18 @@ ChefSpec::Coverage.start! { add_filter '<%= cookbook_name %>' }
 CENTOS_7_OPTS = {
   platform: 'centos',
   version: '7.2.1511',
```
Comment (ramereth) [inline templates/default/spec_helper.rb.erb]: Remove comma

File: templates/default/spec_helper.rb.erb [hunk 3]
```diff
@@ -6,17 +6,18 @@ ChefSpec::Coverage.start! { add_filter '<%= cookbook_name %>' }
...
-  log_level: :fatal
 }.freeze
 
 CENTOS_6_OPTS = {
   platform: 'centos',
   version: '6.8',
```
Comment (ramereth) [inline templates/default/spec_helper.rb.erb]: Remove comma

File: templates/default/spec_helper.rb.erb [hunk 4]
```diff
@@ -6,17 +6,18 @@ ChefSpec::Coverage.start! { add_filter '<%= cookbook_name %>' }
...
-  log_level: :fatal
 }.freeze
 
 DEBIAN_8_OPTS = {
   platform: 'debian',
   version: '8.4',
```
Comment (ramereth) [inline templates/default/spec_helper.rb.erb]: Remove comma

File: templates/default/spec_helper.rb.erb [hunk 5]
```diff
@@ -5,18 +5,19 @@ ChefSpec::Coverage.start! { add_filter '<%= cookbook_name %>' }
...
-  version: '7.2.1511',
-  log_level: :fatal
+  version: '7.2.1511'
 }.freeze
 
 CENTOS_6_OPTS = {
   platform: 'centos',
-  version: '6.8',
-  log_level: :fatal
+  version: '6.8'
```
Comment (ramereth) [inline templates/default/spec_helper.rb.erb]: Please switch this to 6.7 so it matches what we have in TK.

**Comment by ramereth** [comment]
:+1:

**Comment by armiller** [comment]
!bump patch

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.0.6.  Have a nice day!

**Pull request merged**